react/.eslintrc.json

```diff
@@ -45,6 +48,11 @@
     "react/prop-types": 2,
     "react/require-render-return": 2,
     "react/self-closing-comp": 2,
-    "react/sort-comp": 2
+    "react/sort-comp": 2,
+
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
+
+    "react/display-name": 0
```

Comment:
probably should move this above the `react-hook` so all the `react` rules are together